src/app/(pages)/projects/page.tsx

```diff
@@ -0,0 +1,16 @@
+"use client";
+ import { useQuery} from 'react-query';
+ import { getAllProjects } from '@/app/Service/ProjectServices';
+import ResponsiveDrawer from '@/app/components/drawer/Drawer';
+
+
+function page() {
+   const { data: Allprojects} = useQuery('Allprojects', getAllProjects);
```

Comment:
you should handle loading and error 
use query handle that for us so get `isLoading` and `error` properties from useQuery and handle this status accordingly